e2e-tests/src/test/committee_split.rs

```diff
@@ -0,0 +1,151 @@
+use std::{thread::sleep, time::Duration};
+
+use aleph_client::{
+    pallets::{elections::ElectionsSudoApi, session::SessionApi},
+    primitives::CommitteeSeats,
+    utility::BlocksApi,
+    waiting::{BlockStatus, WaitingExt},
+    AccountId, AsConnection, Pair, SignedConnection, TxStatus,
+};
+use log::info;
+
+use crate::{
+    accounts::{get_validator_seed, get_validators_keys, NodeKeys},
+    config::setup_test,
+    rewards::set_invalid_keys_for_validator,
+};
+
+/// time needed for 5 out of 7 block producers to do 3 sessions.
+const SLEEP_DURATION: Duration = Duration::from_secs(126);
+
+async fn prepare_test() -> anyhow::Result<()> {
+    let config = setup_test();
+
+    let accounts = get_validators_keys(config);
+    let connection = config.create_root_connection().await;
+
+    let new_validators: Vec<AccountId> = accounts
+        .iter()
+        .map(|pair| pair.signer().public().into())
+        .collect();
+
+    let seats = CommitteeSeats {
+        reserved_seats: 3,
+        non_reserved_seats: 4,
+        non_reserved_finality_seats: 1,
+    };
+
+    connection
+        .change_validators(
+            Some(new_validators[0..3].to_vec()),
+            Some(new_validators[3..].to_vec()),
+            Some(seats),
+            TxStatus::InBlock,
+        )
+        .await?;
+
+    Ok(())
+}
+
+fn validator_address(index: u32) -> String {
```

Comment:
Huh, we don't have such a function already? If so, could you put it in some common tools, since it seems very generically useful?

Reply:
done